manifests/init.pp

```diff
@@ -4,10 +4,14 @@
 #
 # @see http://www.freedesktop.org/software/polkit/docs/latest/ PolicyKit Documentation
 #
+# @param package_ensure The ensure status of packages
+#
 # @author Trevor Vaughan <mailto:tvaughan@onyxpoint.com>
 #
-class polkit {
+class polkit (
+  Polkit::PackageEnsure $package_ensure = 'installed'
```

Comment:
@jeefberkey SIMP-3035 completed the catalyst so this needs to be updated to use the new catalyst.